### PR TITLE
Node instead of shell

### PR DESCRIPTION
**Pull request by icarusz**
This PR updates the front page to remove the shell hello world and move it to node as well as to change to a video of those commands (replacing static old code).  
It also has a bunch of needed revisions to the notebook tutorial to make that work.
Last it re-routes the user a bit (away from old wasm video) to other wasm examples.

Hopefully all ready to merge as soon as we update to g4 as primary download version.

Dont merge before that!

### CODE REVIEW

File: tutorials/node.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+To show how easy Tupelo is to use we are going to create our first ChainTree and then add
+data to it ("Hello World" of course) and have it signed by the Tupelo TestNet.  
+
+We are going to accomplish this by using the Tupelo WASM SDK and a node repl session.
+
+First you will need to use [npm](https://www.npmjs.com/get-npm) to install two dependencies.
+
+```
+npm install tupelo-wasm-sdk@0.6.0-rc3
+npm install multicodec@0.5.6
```
Comment (brandonwestcott) [inline tutorials/node.md]: seems odd that this isn't explicitly installed with the sdk, I'll look into it

File: tutorials/node.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+```
+> await tree.resolveData("path")
+> await tree.id()
+```
+
+## Commentary
+Read below if you'd like a (brief) explanation of what each of the above steps does.
+
+1. setup a connection to the default community and Tupelo TestNet
```
Comment (brandonwestcott) [inline tutorials/node.md]: Capitalize `Setup` given the other bullets are capitalized

File: tutorials/node.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+3. Create a new empty tree with the new keypair
+```
+> const tree = await sdk.ChainTree.newEmptyTree(community.blockservice, key) 
+```
+
+4. Play a transaction on the TestNet -- Essentially add data to your Chaintree and get it signed by the network.
+```
+> let resp = await community.playTransactions(tree, [sdk.setDataTransaction("path", true)])
```
Comment (brandonwestcott) [inline tutorials/node.md]: Maybe its intentional here for brevity, but the value in this transaction is different than up top (`true` vs `"Hello Decentralized World!"`)

**Review by brandonwestcott — APPROVED** [review]
Looks great to me with a few minor comments.

I actually really like the new node repl tutorial even more than the old one.

File: tutorials/node.md (hunk 1 above)
Comment (icarusz) [reply tutorials/node.md]: It was that the latest multicodec broke things.  May be fixed by now.

**Comment by tobowers** [comment]
it should be fixed. can you test?

**Pull request merged**